otx/algorithms/detection/tasks/inference.py

```diff
@@ -226,28 +229,33 @@ def _init_recipe(self):
         if train_type not in (TrainType.SEMISUPERVISED, TrainType.INCREMENTAL):
```

Comment:
How about declaring it at the top of the file and using it?
`SUPPORTED_TRAIN_TASK = [TrainType.SEMISUPERVISED, TrainType.INCREMENTAL]`
&
`if train_type not in SUPPORTED_TRAIN_TASK:`

Reply:
It would be better, let me modify it. Thanks